tests/test_workflow_fmriprep.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+import pytest
+from .conftest import global_config_for_testing
+
+import nipoppy.workflow.logger as my_logger
+import nipoppy.workflow.proc_pipe.fmriprep.run_fmriprep as fmriprep_module
+from nipoppy.workflow.proc_pipe.fmriprep.run_fmriprep import run, run_fmriprep
+
+
+def create_dummy_fs_license(pth: Path) -> None:
+    pth = pth / "freesurfer"
+    pth.mkdir(parents=True, exist_ok=True)
+    license_file = pth / "license.txt"
+    license_file.write_text("dummy content")
+
+
+def dummy_bids_filter_file() -> Path:
+    """TODO probably don't want the bids filter file to be in the module directory"""
+    pth = Path(fmriprep_module.__file__).parent
+    return pth / "bids_filter.json"
```

Comment:
the default path for the bids_filter.json will probably have to be changed because if someone install the package with pip install but not in editable mode, it will be hard for this to work.